### PR TITLE
fix: COM-3056 Horizontal scroll fix

### PR DESCRIPTION
**Pull request by dyurein**
* jira release board: https://jira.csssr.io/projects/COM/versions/12703
* testing: http://release-blog-year-results.csssr.cloud

## Checklist
- [x] PR is directed to the release branch (if there is a release branch, otherwise this item can be removed)
- [ ] Code meets requirements not only from the description of the task, but also from its comments
- [x] Code works correctly on all resolutions
- [ ] Task completed with comments for testing (if necessary, otherwise this item can be removed)
- [x] Browser dev tools has no errors
- [ ] Got acquainted with the project documentation and updated it if necessary

### CODE REVIEW

File: components/Newsletter/Newsletter.jsx [hunk 1]
```diff
@@ -34,10 +35,11 @@ const Newsletter = ({ className, kind, BENCHMARK_EMAIL_TOKEN, BENCHMARK_EMAIL_LI
           <PictureSmart
             requireImages={kind === 'post' ? postImg : img}
             className="picture"
-            alt={imgAlt}
+            alt={imgAlt}f
           />
         </div>
       </Grid>
+      <Global styles={mainStyles} />
```
Comment (ArtyomResh) [inline components/Newsletter/Newsletter.jsx:42]: Думаю тут можешь просто в общих стилях на `.main` добавить `overflow: hidden;`, `Global` тут не особо нужен

File: components/Newsletter/Newsletter.jsx [hunk 2]
```diff
@@ -34,10 +35,11 @@ const Newsletter = ({ className, kind, BENCHMARK_EMAIL_TOKEN, BENCHMARK_EMAIL_LI
           <PictureSmart
             requireImages={kind === 'post' ? postImg : img}
             className="picture"
-            alt={imgAlt}
+            alt={imgAlt}f
```
Comment (ArtyomResh) [inline components/Newsletter/Newsletter.jsx:38]: F 
:D

**Review by ArtyomResh — CHANGES_REQUESTED** [review]

File: components/Newsletter/Newsletter.jsx (hunk 2 above)
Comment (dyurein) [reply components/Newsletter/Newsletter.jsx:38]: **F**

Comment (dyurein) [reply components/Newsletter/Newsletter.jsx]: Исправил

**Review by ArtyomResh — APPROVED** [review]

**Pull request merged**